main/main.cpp

```diff
@@ -3063,20 +3063,20 @@ bool Main::start() {
 			real_t stretch_scale = GLOBAL_GET("display/window/stretch/scale");
 
 			Window::ContentScaleMode cs_sm = Window::CONTENT_SCALE_MODE_DISABLED;
-			if (stretch_mode == "canvas_items") {
+			if (stretch_mode == "Canvas Items") {
```

Comment:
Maybe use something like this, to avoid compatibility breaking:
```suggestion
			if (stretch_mode.to_snake_case() == "canvas_items") {
```

Reply:
Already thought of that, but it won't fix this:

![image](https://github.com/godotengine/godot/assets/30739239/13598dd2-db0f-41c0-afab-bb9e58c97520)